react-components/src/analysis/view/AnalysesView.js

```diff
@@ -0,0 +1,955 @@
+import TableRow from "@material-ui/core/TableRow";
+import { withStyles } from "@material-ui/core/styles";
+
+import LaunchIcon from "@material-ui/icons/Launch";
+import ListAltIcon from "@material-ui/icons/ListAlt";
+
+
+function AnalysisName(props) {
+    const name = props.analysis.name;
+    const isBatch = props.analysis.batch;
+    const className = props.classes.analysisName;
+    const handleGoToOutputFolder = props.handleGoToOutputFolder;
+    const handleBatchIconClick = props.handleBatchIconClick;
+    const interactiveUrls = props.analysis.interactive_urls;
+    const handleInteractiveUrlClick = props.handleInteractiveUrlClick;
+    const status = props.analysis.status;
+
+    if (isBatch) {
+        return (
+            <span className={className} onClick={handleGoToOutputFolder}>
+                <ListAltIcon onClick={handleBatchIconClick}
+                             style={{color: Color.darkGreen}}/>
+                <sup>{name}</sup>
+            </span>
+        );
+    } else if (status === analysisStatus.RUNNING && interactiveUrls && interactiveUrls.length > 0) {
+        return (
+            <span className={className} onClick={handleGoToOutputFolder}>
+                <LaunchIcon onClick={() => handleInteractiveUrlClick(interactiveUrls[0])}
+                            style={{color: Color.darkBlue}}/>
+                <sup>{name}</sup>
+            </span>
+        )
+    } else {
+        return (
+            <span className={className} onClick={handleGoToOutputFolder}>
+                {name}
+            </span>
+        );
+    }
+}
+
+function AppName(props) {
+    const analysis = props.analysis;
+    const name = analysis.app_name;
+    const isDisabled = analysis.app_disabled;
+    const className = props.classes.analysisName;
+    const handleRelaunch = props.handleRelaunch;
+
+    if (!isDisabled) {
+        return (
+            <span className={className} onClick={() => handleRelaunch(analysis)}>
+                {name}
+            </span>
+        );
+    } else {
+        return (
+            <span>
+                {name}
+            </span>
+        );
+    }
+
+}
+
+function Status(props) {
+    const {analysis, onClick, username} = props;
+    if (username === analysis.username && analysis.status !== analysisStatus.CANCELED) {
+        return (<DEHyperLink
+            onClick={(analysis) => onClick(analysis)}
+            text={analysis.status}/>);
+    } else {
+        return (<span style={{textAlign: 'left', fontSize: '11px'}}>{analysis.status}</span>);
+    }
+}
+
+const columnData = [
+    {name: "Name", numeric: false, enableSorting: true},
+    {name: "Owner", numeric: false, enableSorting: true},
+    {name: "App", numeric: false, enableSorting: true},
+    {name: "Start Date", numeric: false, enableSorting: true},
+    {name: "End Date", numeric: false, enableSorting: true},
+    {name: "Status", numeric: false, enableSorting: true},
```

Comment:
I noticed all of our tables so far have column headers where the text is like this.  I'm wondering if there's some way to still get the text through i18n. I haven't dug into it much myself though, not necessary for this PR